### PR TITLE
Active Directory Query v2 - global catalog reference

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/XSUP-44308

## Description
Added documentation on global catalog for queries and operations across multiple domains by using port 3268

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ configuration:
   type: 0
   required: true
   section: Connect
-- display: Port. If not specified, the default port is 389, or 636 for LDAPS.
+- display: Server port. If not specified, the default port is 389, 636 for LDAPS, or 3268 for global catalog servers.
```
Comment (ShacharKidor) [inline Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml:17]:
```suggestion
- display: Server port. If not specified, the default port is 389 for LDAP, 636 for LDAPS, or 3268 for global catalog servers.
```

File: Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Use Cases
 | **Parameter** | **Description**                                                                                  | **Required** |
 |--------------------------------------------------------------------------------------------------| --- | --- |
 | Server IP address (for example, 192.168.0.1) | The Server IP that should be used to access Active Directory.                                    | True |
-| Port  | Server port. If not specified, the default port is 389, or 636 for LDAPS.                        | False |
+| Port  | Server port. If not specified, the default port is 389, 636 for LDAPS, or 3268 for global catalog servers.                        | False |
```
Comment (ShacharKidor) [inline Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md:48]:
```suggestion
| Port  | Server port. If not specified, the default port is 389 for LDAP, 636 for LDAPS, or 3268 for global catalog servers.                        | False |
```

File: Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md [hunk 3]
```diff
@@ -59,7 +59,8 @@ Use Cases
 | Group CN for terminated employees |                                                                                                  | False |
 | Create user if does not exist | If true, the user is created if the user profile doesn't exist in AD. Used in IAM commands only. | False |
 
-
+> <i>Note:</i> For queries and operations across multiple domains within a forest the server port should be 3268. This port is used for queries specifically targeted for the global catalog. LDAP requests sent to port 3268 can be used to search for objects in the entire forest.
+For more information on global catalog see the [Microsoft documentation](https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/plan/planning-global-catalog-server-placement).
```
Comment (ShacharKidor) [inline Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md:63]:
```suggestion
> <i>Note:</i> For queries and operations across multiple domains within an Active Directory forest the server port should be 3268. This port is used for queries specifically targeted for the global catalog. LDAP requests sent to port 3268 can be used to search for objects in the entire Active Directory forest.
For more information on global catalog see the [Microsoft documentation](https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/plan/planning-global-catalog-server-placement).
```

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
@adi88d Nice work, 
Please add a tech writer for a docs review.

File: Packs/Active_Directory_Query/ReleaseNotes/1_6_44.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Active Directory Query v2
+
+- Documentation and metadata improvements.
```
Comment (ShirleyDenkberg) [inline Packs/Active_Directory_Query/ReleaseNotes/1_6_44.md:6]:
```suggestion
Documentation and metadata improvements.
```
Do you need to be more specific?  Added 3268 for global catalog servers port value.

**Comment by ShirleyDenkberg** [comment]
@ShacharKidor Doc review completed.

**Review by ShacharKidor — APPROVED** [review]

**Pull request merged**